pmultiqc_service/app.py

```
@@ -1374,9 +1375,12 @@ def create_zip_report(output_path: str, zip_path: str) -> bool:
             return False
         
         files_found = []
+        zip_filename = os.path.basename(zip_path)
         for root, dirs, files in os.walk(output_path):
```

Ruff (0.12.2)

Loop control variable `dirs` not used within loop body

Rename unused `dirs` to `_dirs`

(B007)